ui/webui/test/README.rst

```diff
@@ -1,17 +1,47 @@
-Integration Tests of Anaconda WebUI
-===================================
+Anaconda Web UI tests
+=====================
 
-This directory contains automated integration tests for Anaconda WebUI, and the support files for them.
+The Web UI tests are based on cockpit's testing framework which in turn is based
+on python's unittest library. There are two kinds of Web UI tests, "integration"
+that test only one specific feature at a time and "end to end" that go through the
+entire installation process including reboot to the installed system. Both types
+use the same library of helper functions, to avoid duplicating code and to make
+the test development easier, but they are executed in different ways because of
+the higher requirements of the end to end tests.
 
+Test development
+----------------
+
+Before test case for testing a new user interface can be written any new elements
```

Comment:
You need either plural or an article here, it's wrong grammatically how it is I think.
aka: 'Before a test case' or 'Before test cases'

Also I think the 'for testing a new user interface' is redundant.